apps/ewallet/lib/ewallet/exporters/csv_exporter.ex

```diff
@@ -61,11 +61,32 @@ defmodule EWallet.CSVExporter do
      }}
   end
 
+  def terminate(reason, %{export: %{status: "processing"} = export}) do
+    {:ok, _} =
+      AdapterHelper.store_error(export, "The server crashed during processing.", inspect(reason))
```

Comment:
Minor nitpick. Can we say "The **export** server crashed ..."? Because it could mean the web server as well.

Or "The **exporter** crashed..."